src/core/src/core_logic/PatchAssessor.py

```diff
@@ -87,6 +90,14 @@ def start_assessment(self):
                     self.status_handler.set_assessment_substatus_json(status=Constants.STATUS_ERROR)
                     raise
 
+        endTime = self.env_layer.datetime.datetime_utcnow()
+        patchService = self.package_manager.__class__.__name__
+        machine_details = TelemetryWriter.machine_info
```

Comment:
Why do we need to print machine info again, since we do log those during TelemetryWriter init. (line 48 in TelemetryWriter: self.write_machine_config_info() logs these)

Reply:
The reason for this is same as I have mentioned in another comment:

Yes, the information which we are logging here are already there in different places but to get all the related information for a job, currently we need join in kusto queries which is not scalable. We are logging all the information in one log so that we don't need join in kusto queries. We need efficient kusto queries because we are going to use kusto queries in future to calculate improvements that we are going to do in the script.

Reply:
Removed original print machine info. Printing machine info as part of stop in Stopwatch class.